tools/mafft/mafft.xml

```diff
@@ -0,0 +1,152 @@
+<tool id="rbc_mafft" name="MAFFT" version="7.221.0">
+    <description>Multiple alignment program for amino acid or nucleotide sequences</description>
+    <requirements>
+        <requirement type="package" version="7.221">mafft</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Error occurred. Please check Tool Standard Error" />
+        <exit_code range=":-1" level="fatal" description="Error occurred. Please check Tool Standard Error" />
+    </stdio>
+    <version_command>
+    <![CDATA[
+        mafft --version
+    ]]>
+    </version_command>
+    <command>
+    <![CDATA[
+        #if $cond_flavour.flavourType != 'custom'
+            $cond_flavour.flavourType
+        #elif $cond_flavour.flavourType == 'custom'
+            ### full parameter options
+            mafft
+        #end if
+        
+        ## specify threads to use
+        --thread \${GALAXY_SLOTS:-1}
+        
+        #if $outputFormat.value == 'clustalw'
+            --clustalout
+        #end if
+        
+        $inputSequences > 
+        
+        #if $outputFormat.value == 'fasta'
```

Comment:
Note this will use an additional copy operation. It's your decision if you want to go for cleaner code or for better performance :)